modules/host/negotiaterenewcontract.go

```diff
@@ -241,7 +241,7 @@ func (h *Host) managedVerifyRenewedContract(so *storageObligation, txnSet []type
 	}
 	// The WindowEnd for the new file contract must be further in the future
 	// than the WindowEnd for the existing file contract.
-	if fc.WindowEnd <= so.proofDeadline() {
+	if fc.WindowEnd < so.proofDeadline() {
```

Comment:
Ran new tests with `<=`, confirmed that fc37f810b8cbf69c5ac7251eb3baaf1543245c68 fixes the problem.